docs/en/sql-reference/data-types/special-data-types/interval.md

```diff
@@ -53,29 +53,28 @@ SELECT now() as current_date_time, current_date_time + INTERVAL 4 DAY
 └─────────────────────┴───────────────────────────────┘
 ```
 
-Intervals with different types can’t be combined. You can’t use intervals like `4 DAY 1 HOUR`. Specify intervals in units that are smaller or equal to the smallest unit of the interval, for example, the interval `1 day and an hour` interval can be expressed as `25 HOUR` or `90000 SECOND`.
```

Comment:
The deleted information mentioned here is still valid, right? E.g. `4 DAY 1 HOUR` still doesn't work.

Reply:
Thanks for the catch! Now I remember to notice this, but forgot to comment on it.